bucket/win11dragdroptaskbarfix.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "2.2.0.0",
+    "description": "Fixes the missing 'Drag & Drop to the Taskbar' support in Windows 11.",
+    "homepage": "https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix",
+    "license": "GPL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/releases/download/v.2.2.0.0-release/Windows11DragAndDropToTaskbarFix.exe",
+            "hash": "9306034e67f3a059427d67655e1e5be36d57f3222250cdb41fb86f560cc585b1"
+        }
+    },
+    "pre_install": [
+        "if (!(Test-Path \"$persist_dir\\Windows11DragAndDropToTaskbarFixConfig.txt\")) {",
+        "   New-Item \"$dir\\Windows11DragAndDropToTaskbarFixConfig.txt\" | Out-Null",
+        "   Set-Content -Path \"$dir\\Windows11DragAndDropToTaskbarFixConfig.txt\" -Value \"//Should the program run automatically on system startup? 1 = true, 0 = false.\nAutomaticallyRunThisProgramOnStartup=1\nStartThisProgramAsAdministrator=0\n\n//For more configuration options, please visit: https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/CONFIGURATION.md\" | Out-Null",
+        "}"
+    ],
+    "shortcuts": [
+        [
+            "Windows11DragAndDropToTaskbarFix.exe",
+            "Windows 11 Drag & Drop to the Taskbar (Fix)"
+        ]
+    ],
+    "persist": [
+        "Windows11DragAndDropToTaskbarFixConfig.txt"
+    ],
```

```suggestion
    "persist": "Windows11DragAndDropToTaskbarFixConfig.txt",
```

Is the application able to read this file correctly (when you create it in pre_install)?

---

Yes, for example if I change `StartThisProgramAsAdministrator=0` to `1`, the program opens as admin on first run.